src/test/kotlin/domain/CardsTest.kt

```diff
@@ -0,0 +1,98 @@
+package domain
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class CardsTest {
+    @Test
+    fun `카드의 개수는 2장 이상이어야 한다`() {
+        // given
+
+        // when
+
+        // then
+        assertThrows<IllegalStateException> { Cards(setOf<Card>(Card(CardCategory.CLOVER, CardNumber.ACE))) }
```

Comment:
given, when, then 항목을 분리해보아요!
```suggestion
// given
val givenCards = setOf<Card>(Card(CardCategory.CLOVER, CardNumber.ACE))
// when, then
assertThrows<IllegalStateException> { Cards(cards) }
```